Makefile

```diff
@@ -18,6 +20,7 @@ b2d_iso: iso_creation
 
 .PHONY: iso_creation
 iso_creation: init
+	@echo "BUILD_ID=${GITTAG}-${DATE}" >> ${CODE_DIR}/os-release
```

Comment:
I guess, we can also use `envsetst` for `NAME` too with value `ISO_NAME`. Also build `PRETTY_NAME` with same.
It will help to maintain these information from one point. Right now, it seems two place of existence for these variables.